docs/solutions/rhel74_image/README.md

```diff
@@ -5,9 +5,10 @@ There are several prerequisites for using this process:
    RH Username and Password to allow you to temporarily subscribe the instance that is building the image 
    and get access to the various RH repos.
 2. The template expects pre-configured VCNs and Subnets.  
-3. You need to have a bucket in the OCI object store that contains the RHEL 7.4 iso.  The bucket must be in 
-   your tenancy, and must be accessible by the user executing the build.
+3. You need to provide a URL that points to the RHEL 7.4 ISO.  This URL must contain the name of the ISO, 
+   with an '.iso' extension.  An OCI PAR works well for this operation.
```

Comment:
Might want to spell out pre-authenticated request and link to https://docs.us-phoenix-1.oraclecloud.com/Content/Object/Tasks/managingobjects.htm#par.

Reply:
Will do.  Will post when complete.

Reply:
Completed.  Changes made.